copilot/fastapi/app/tvp.py

```
@@ -0,0 +1,394 @@
+    Status of TVP GitOps reconciliation.
+    
+    Attributes:
+        is_reconciling (bool): Whether reconciliation is currently in progress.
+        last_reconciliation (Optional[str]): ISO formatted timestamp of the last reconciliation.
+        status (str): Current status of the TVP service ("active" or "inactive").
+        applications (List[Dict[str, Any]]): List of applications managed by TVP.
+    """
+    is_reconciling: bool
+    last_reconciliation: Optional[str] = None
+    status: str
+    applications: List[Dict[str, Any]] = []
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)